addons/l10n_mx/data/template/account.tax-mx.csv

```diff
@@ -75,19 +75,27 @@
 "","","","","","","","","","","","","","","tax","invoice","cuenta216_10_20","-DIOT: Retención","",""
 "","","","","","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","","","","","tax","refund","cuenta216_10_20","+DIOT: Retención","",""
-"tax13","19","0%","","VAT(0%)","0.0","percent","purchase","tax_group_iva_0","on_payment","cuenta119_01","","Tasa","","base","invoice","","+DIOT: 0%","","IVA(0%)"

```

Comment:
Maybe we could leave the sequence like they were to keep the github history :smile: